README.md

```diff
@@ -10,39 +10,32 @@ Helm charts to install Istio and riff.
 
 - a running kubernetes cluster (1.14+)
 - kubectl (1.14+)
-- helm (2.13+)
+- helm (3.0+)
 
 ### Steps
 
-1. Initialize Helm
-
-   ```sh
-   kubectl create serviceaccount tiller -n kube-system
-   kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount kube-system:tiller
-   helm init --wait --service-account tiller
-   ```
-
 1. Load the projectriff charts
 
    ```sh
    helm repo add projectriff https://projectriff.storage.googleapis.com/charts/releases
    helm repo update
    ```
 
-1. Install Istio (optional, required for the Knative runtime)
+2. Install Istio (optional, required for the Knative runtime)
```

Comment:
`1.` is the preferred pattern for markdown so that you don't need to update every item in the list when there is an insertion/removal/reordering

Reply:
I don't think I renumbered this, maybe my code editor changed the numbering